.travis.yml

```diff
@@ -9,6 +9,8 @@ install:
   - go get -v github.com/mattn/goveralls
 
 script:
+  # This unencryptes envvars.bash.enc into envvars.bash so that we can run the tests
+  - openssl aes-256-cbc -K $encrypted_263604b5b445_key -iv $encrypted_263604b5b445_iv -in envvars.bash.enc -out envvars.bash -d
```

Comment:
Hello @myleshorton, do you have a copy of these keys? I wanted to test `peerscanner/cf` and I guess the required env vars are here.